posthog/temporal/data_imports/sources/shopify/settings.py

```diff
@@ -0,0 +1,19 @@
+from posthog.temporal.data_imports.sources.shopify.constants import (
+    ABANDONED_CHECKOUTS,
+    ARTICLES,
+    SHOPIFY_PAYMENTS_BALANCE_TRANSACTIONS,
+)
+from posthog.warehouse.types import IncrementalField
+
+INCREMENTAL_FIELDS: dict[str, list[IncrementalField]] = {
+    ABANDONED_CHECKOUTS: [
+        # {
+        #     "label": "created_at",
+        #     "type": IncrementalFieldType.DateTime,
+        #     "field": "created",
+        #     "field_type": IncrementalFieldType.Integer,
+        # }
+    ],
+    ARTICLES: [],
+    SHOPIFY_PAYMENTS_BALANCE_TRANSACTIONS: [],
+}
```

Comment:
Any reason we're not adding these now?

Reply:
just trying to keep the PR small for review. added them in. looking at whether i need to modify the external_table_definitions file now.